package.json

```diff
@@ -4,12 +4,12 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
-    "start": "npx babel-node src/index.js",
+    "build": "babel -d build src",
+    "start": "npm run build && node build/index.js",
```

Comment:
Wouldn't using `babel-node` be better? Because it won't generate build source during `dev`.

Reply:
It's slow and sometimes doesn't work as expected but hard to debug.

Reply:
Ok. I haven't realized that but I trust you 👍.